src/config/main.js

```diff
@@ -29,10 +29,20 @@ const audioCodes = {
 const MTURK = (!jsPsych.turk.turkInfo().outsideTurk)
 const AT_HOME = (process.env.REACT_APP_AT_HOME === 'true')
 const VIDEO =  (process.env.REACT_APP_VIDEO === 'true')
+const ONLINE = (process.env.REACT_APP_ONLINE=== 'true')
```

Comment:
`ONLINE` doesn't seem to be used anywhere - is it needed?

Reply:
I think this relates more to what Galen is doing and I'm not sure if it is necessary yet, so I think we could keep it for now.